akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -325,12 +325,12 @@ class TestKit(system: ActorSystem) {
   /**
    * Same as `expectNoMsg(remainingOrDefault)`, but correctly treating the timeFactor.
    */
-  def expectNoMsg(): Unit = tp.expectNoMsg()
+  def expectNoMsg(): Unit = tp.expectNoMessage()
 
   /**
    * Assert that no message is received for the specified time.
    */
-  def expectNoMsg(max: FiniteDuration): Unit = tp.expectNoMsg(max)
+  def expectNoMsg(max: FiniteDuration): Unit = tp.expectNoMessage(max)
```

Comment:
`expectNoMsg` and `expectNoMessage` are not completely equivalent: `expectNoMsg` dilated the duration, while `expectNoMessage` does not (#20255).

Perhaps we should deprecate `expectNoMsg` in the javadsl as well, and introduce `expectNoMessage` here, too?

Reply:
:+1: